fcrepo-import-export-driver/src/test/java/org/fcrepo/importexport/integration/ExecutableJarIT.java

```diff
@@ -0,0 +1,113 @@
+import static org.junit.Assert.assertTrue;
+import static org.slf4j.LoggerFactory.getLogger;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.io.IOUtils;
+import org.fcrepo.client.FcrepoClient;
+import org.fcrepo.client.FcrepoOperationFailedException;
+import org.fcrepo.client.FcrepoResponse;
+import org.fcrepo.exporter.ArgParser;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+/**
+ * @author awoods
+ * @since 2016-09-01
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration("/spring-test/test-container.xml")
+public class ExecutableJarIT extends AbstractResourceIT {
+
+    private static final Logger logger = getLogger(ExecutableJarIT.class);
+
+    private URI url;
+
+    private static final String EXECUTABLE = System.getProperty("fcrepo.executable.jar");
+    private static final String TARGET_DIR = System.getProperty("project.build.directory");
+
+    public ExecutableJarIT() throws Exception {
+        super();
+        client = FcrepoClient.client().build();
+    }
+
+    @Before
+    public void before() {
+        url = URI.create(serverAddress + UUID.randomUUID());
+        assertNotNull(EXECUTABLE);
+        assertNotNull(TARGET_DIR);
+    }
+
+    @Test
+    public void testJarSanity() throws IOException, InterruptedException {
+        // Run the executable jar with no arguments
+        final ProcessBuilder builder = new ProcessBuilder("java", "-jar", EXECUTABLE);
+        builder.directory(new File(TARGET_DIR));
+        final Process process = builder.start();
+        logger.debug("Output of jar run: {}", IOUtils.toString(process.getInputStream()));
+
+        // Verify it ran
+        assertTrue("Process did not exit before timeout!", process.waitFor(1000, TimeUnit.SECONDS));
+        assertEquals("Did not exit with success status!", 0, process.exitValue());
+    }
+
+    @Test
+    public void testJarExport() throws Exception {
+        // Create a repository resource
+        final FcrepoResponse response = create();
+        assertEquals(CREATED.getStatusCode(), response.getStatusCode());
+        assertEquals(url, response.getLocation());
+
+        // Run an export process
+        final ProcessBuilder builder = new ProcessBuilder("java", "-jar", EXECUTABLE,
+                "-m", "export",
+                "-d", TARGET_DIR,
+                "-r", url.toString());
+
+        builder.directory(new File(TARGET_DIR));
+        final Process process = builder.start();
+        logger.debug("Output of jar run: {}", IOUtils.toString(process.getInputStream()));
+
+        // Verify
+        assertTrue("Process did not exit before timeout!", process.waitFor(1000, TimeUnit.SECONDS));
```

Comment:
As someone who is going to write more integration tests, the logic in lInes 91 through 103 should probably be a helper method.  You can already reuse that method in the other test.  This code will likely be used in almost every new test.  Then if we find we want to include stderr output in our logs (or something else), we'll only have to update one place.

Whether to include this assertion (or the next) is up to you... I can see how it might be messy to have assertions in a helper method, and I can envision tests in the future where we WANT an non-zero exitValue from the process.